variables/_z-index.pcss

```
@@ -4,4 +4,5 @@
 	 * ----------------------------------------------------------------------------- */
 
 	--z-index-spinner-container: 100;
+	--z-index-calendar-multiday: 99;
```

```suggestion
	--z-index-calendar-multiday: 95;
```

Allowing breathing room